chartpress.py

```diff
@@ -292,6 +294,38 @@ def _strip_identifiers_build_suffix(identifier):
     return re.sub(r'[-\.]\d{3,}\.\w{4,}\Z', "", identifier)
 
 
+def _get_published_chart_information(chart_repo_url, chart_name, chart_version):
+    """
+    Provide information about a chart release from a chart repository url along
+    with a chart name and version to search for.
+
+    - Returns the dictionary representing the matching chart entry from the Helm
+      chart repository if a matching entry is found.
+    - Returns None if a Helm chart repostiory is found but the matching chart
+      name and version isn't.
+    - Raises errors if the Helm chart repository isn't accessible or doesn't
+      seem to be one.
+    """
+    index_url = urljoin(chart_repo_url + "/", "index.yaml")
```

Comment:
urljoin is what adds the slash here

```suggestion
    index_url = urljoin(chart_repo_url, "index.yaml")
```

Reply:
I was very confused by this as well. I still don't get this properly. But, if I remove it my tests fail when this function is called, and here is some debugging information of relevance.

Adding some print statements:
```python
    print(chart_repo_url)
    # https://jupyterhub.github.io/helm-chart

    print(urljoin(chart_repo_url, "index.yaml"))
    # https://jupyterhub.github.io/index.yaml

    print(urljoin(chart_repo_url + "/", "index.yaml"))
    # https://jupyterhub.github.io/helm-chart/index.yaml
```

Reply:
```ipython
In [4]: urllib.parse.urljoin("https://example.local/ignored_subpath", "index.yaml")
Out[4]: 'https://example.local/index.yaml'

In [5]: urllib.parse.urljoin("https://example.local/not_ignored_subpath/", "index.yaml")
Out[5]: 'https://example.local/not_ignored_subpath/index.yaml'

In [6]: urllib.parse.urljoin("https://example.local/not_ignored_subpath//", "index.yaml")
Out[6]: 'https://example.local/not_ignored_subpath/index.yaml'
```